wrangler-transform/widgets/Wrangler-transform.json

```diff
@@ -77,6 +104,32 @@
   ],
   "emit-alerts": true,
   "emit-errors": true,
+  "filters": [
+    {
+      "name": "PreconditionValueJEXL",
+      "condition": {
+        "expression": "preconditionLanguage == 'jexl'"
```

Comment:
Re-iterating what @fernst mentioned earlier, when upgrades are done via scripts and not through the UI. We cannot assume this property will be set since it will not be present in older pipelines. When `null` we should assume it to be `jexl`.